newrelic.js

```diff
@@ -7,16 +7,26 @@
 
 var config = require('./server-config-defaults.js');
 
-if(config.core.newrelic.key) exports.config = {
-	app_name : config.core.newrelic.name || "Scrollback",
-	license_key : config.core.newrelic.key,
-	logging : {
-		level : 'info'
-	},
-	rules: {
-		ignore: [
-			'^/socket/\\*',
-		]
-	}
-};
+if(config.core.newrelic.key) {
+		exports.config = {
+			app_name : config.core.newrelic.name || "Scrollback",
+			license_key : config.core.newrelic.key,
+			logging : {
+				level : 'info'
+			},
+			rules: {
+				ignore: [
+					'^/engine.io'
+				],
+				name: [
+					{ pattern: "/me",   name: "/me"  },
+					{ pattern: "/i/.*", name: "/i/*" },
+					{ pattern: "/s/.*", name: "/s/*" },
+					{ pattern: "/r/.*", name: "/r/*" },
+					{ pattern: /^\/([\w\-]{3,32})\/.*/, name: "/:room/:discussion" },
+					{ pattern: /^\/([\w\-]{3,32})?$/, name: "/:room" }
```

Comment:
I think the second one should be ` /^\/([\w\-]{3,32})\/?$/`. The question mark means "optional" (zero or one instances of previous character), I had added it for handling both `/bangalore` and `/bangalore/` as `/:room`. It needs to have a \/ before it so that it is making the slash optional. Your regex makes the whole `([\w\-]{3,32})` optional, which we don't want.